example/src/main.rs

```diff
@@ -31,9 +32,9 @@ impl SinWave {
 }
 
 impl jack::JackHandler for SinWave {
-    fn process(&mut self, n_frames: u32) -> jack::JackControl {
+    fn process(&mut self, process_scope: &mut jack::ProcessScope) -> jack::JackControl {
         // Get output buffer
-        let out: &mut [f32] = unsafe { self.out_port.as_slice_mut(n_frames) };
+        let out: &mut [f32] = self.out_port.output_buffer(process_scope);
```

Comment:
I can add it in next weekend. Is there a particular thing you want to use this lib for?